content/docs/install/helm-install/index.md

```diff
@@ -116,7 +116,7 @@ to port `8080` on the Keptn API Gateway service in the cluster.
      `<access-option>` must be `LoadBalancer`, `NodePort`, or `ClusterIP`.
      See [Choose access options](../access/) for details.
    * `--create-namespace` -- creates the `keptn` namespace if it does not already exist.
-   * `--set=continuousDelivery.enabled=true` -- install Continuous Delivery support
+   * `--set=bridge.installationType=CONTINUOUS_DELIVERY` -- install Continuous Delivery support
```

Comment:
This setting is actually in the default values anyways, so there's no need to set anything here. We can just remove that `--set`
```suggestion
```